client/question/services/code_evaluators/PythonCodeRunnerServiceSpec.js

```diff
@@ -254,4 +343,62 @@ describe('PythonCodeRunnerService', function() {
       expect(codeEvalResult.getErrorInput()).toEqual(null);
     });
   });
+
+  describe('_standardizeServerOutput', function() {
+    it('should work for task completion stdOut', function() {
```

Comment:
maybe be a bit more specific here -- "should correctly display stdOut on task completion" even, is better.